capabilities-db/mysql.config

```diff
@@ -0,0 +1,4 @@
+[mysql]
```

Comment:
This shouldn't be in the repo!!! Make sure you add "mysql.config" into the .gitignore